data/config.yml

```diff
@@ -816,3 +816,10 @@ migration analytics:
     "Migration Analytics - Button: Create":
       selectors:
         - '{} a:contains("Create report")'
+
+application services (kafka):
+  color: 7
+  pages:
+    OpenShift streams:
```

Comment:
Should this one say something like "Landing page?"
The current would make a thing like:
_ application services (kafka) - OpenShift streams
Is that not kinda redundant?

I would consider doing something like this I think:
```yaml
kafka:
  pages:
    Landing:
      url_rules:
        - /application-services/openshift-streams/
```

Just like we do with Advisor I think.
We dont say 
_insights (advisor) - Insights Advisor - Landing Page
Because that is verbose and redundant.

Thouhts?

Reply:
Got it - sure! You know product better than I'll do. I'll make the change to your proposed: 

  pages:
    Landing:
      url_rules:
        - /application-services/openshift-streams/